<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="acer" failures="0" tests="5" name="Default test" time="0.454" errors="0" timestamp="2023-02-12T11:13:27 IST">
  <testcase classname="assignment12.TestNGTask" name="@BeforeClass setUp" time="0.152">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.openqa.selenium.WebDriver.manage()&amp;quot; because &amp;quot;helper.Utility.driver&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.manage()" because "helper.Utility.driver" is null
at helper.Utility.startBrowser(Utility.java:84)
at assignment12.TestNGTask.setUp(TestNGTask.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase classname="assignment12.TestNGTask" name="loginScenario" time="0.0">
    <skipped/>
  </testcase> <!-- loginScenario -->
  <testcase classname="assignment12.TestNGTask" name="clickManageCourse" time="0.0">
    <skipped/>
  </testcase> <!-- clickManageCourse -->
  <testcase classname="assignment12.TestNGTask" name="clickAddNewCourse" time="0.0">
    <skipped/>
  </testcase> <!-- clickAddNewCourse -->
  <testcase classname="assignment12.TestNGTask" name="addNewCourseForm" time="0.005">
    <skipped/>
  </testcase> <!-- addNewCourseForm -->
  <testcase classname="assignment12.TestNGTask" name="verifyCourse" time="0.0">
    <skipped/>
  </testcase> <!-- verifyCourse -->
</testsuite> <!-- Default test -->
